include/wx/qt/textentry.h

```diff
@@ -36,6 +36,10 @@ class WXDLLIMPEXP_CORE wxTextEntry : public wxTextEntryBase
     virtual bool IsEditable() const override;
     virtual void SetEditable(bool editable) override;
 
+    // For internal use only, see implementation for explanation.
+    // Only controls with wxTE_PROCESS_ENTER flag use this function.
+    void ToggleDefaultButton();
```

Comment:
Let's use port-specific prefix for non-portable functions:
```suggestion
    void QtToggleDefaultButton();
```